src/tox_lsr/config_files/tox-default.ini

```diff
@@ -314,3 +314,16 @@ deps =
 commands =
     curl -L -o {envdir}/report-modules-plugins.py https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/master/report-modules-plugins.py
     python {envdir}/report-modules-plugins.py --details .
+
+[testenv:ansible-managed-var-comment]
+changedir = {toxinidir}
+basepython = python3
+whitelist_externals = grep
```

Comment:
add `echo` to `whitelist_externals` because of https://github.com/linux-system-roles/kdump/runs/4118049578?check_suite_focus=true
```
WARNING: test command found but not installed in testenv
  cmd: /usr/bin/echo
  env: /home/runner/work/kdump/kdump/.tox/ansible-managed-var-comment
Maybe you forgot to specify a dependency? See also the allowlist_externals envconfig setting.
```

Reply:
you also need to end each line with a `\` continuation character - otherwise, tox thinks each line is a separate command